manager/src/main/java/com/redhat/service/smartevents/manager/api/models/responses/ErrorResponse.java

```diff
@@ -0,0 +1,53 @@
+package com.redhat.service.smartevents.manager.api.models.responses;
+
+import org.eclipse.microprofile.openapi.annotations.media.Schema;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@Schema(name = "ErrorResponse",
+        description = "General error response",
+        allOf = { ObjectReference.class, ErrorResponse.class })
+
+public class ErrorResponse extends ObjectReference {
```

Comment:
Isn't this the same of [`ErrorResponse`](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/main/infra/src/main/java/com/redhat/service/smartevents/infra/api/models/responses/ErrorResponse.java)? In general we should avoid to have more classes with same name because the openAPI renames it as `ErrorResponse1`